### ISSUE TITLE
RuntimeWarning: invalid value encountered in log

### ISSUE THREAD

**Issue by jchodera** [choderalab/perses#171]
```
/Users/choderaj/miniconda/lib/python2.7/site-packages/perses-0.1-py2.7.egg/perses/rjmc/geometry.py:287: RuntimeWarning: invalid value encountered in log
  logp_proposal += logp_r + logp_theta + logp_phi + np.log(detJ)
```

**Comment by pgrinaway** [comment]
Will take a look right after getting the AA test in.

**Comment by pgrinaway** [comment]
I can't reproduce this error.

**Comment by steven-albanese** [comment]
I've encountered this same warning when using the script I just pushed to PR #201 

I'm going to try and see if the error is reproducible now

**Comment by jchodera** [comment]
@pgrinaway : It might be stochastic. Maybe you could instrument the code a bit by `catch`ing `RuntimeError`s and printing some info about `detJ` and its calculation when this happens?

**Comment by pgrinaway** [comment]
Resolved in #202
